frontend/main.py

```diff
@@ -0,0 +1,32 @@
+import click
+
+@click.command()
+
+def main():
+    click.clear()
+    venue_decided = click.prompt("Venue Decided? (y/n)")
+    if(venue_decided=='y'):
+        meetup_link = click.prompt("Meetup Link")
+        poster_link = click.prompt("Poster Link")
+        telegram_user = click.prompt("Telegram Username")
+        talk = click.prompt("Talks Decided? (y/n)")
+        if(talk=='y'):
+            click.echo("\nProcessing ↑ (saving to .birdman/<date>/meetup.txt)")
+            click.echo("\nJoin us for the next ILUG-D Meetup on <Date/Time> at <Venue>")
```

Comment:
Take the 'Join us for the next ILUG-D Meetup on <Date/Time> at <Venue>'  in a string and edit it using fstring. https://realpython.com/python-f-strings/